hugo-modules/core/utils/get-page.html

```diff
@@ -12,7 +12,7 @@
 {{ $content_type := .content_type }}
 {{ $page := dict }}
 {{ if and $content_type $entry_id }}
-  {{ if (eq $content_type "page") }}
+  {{ if hasPrefix $content_type "t-" }}
```

Comment:
@bezoerb Almost all adjustments in `hugo-modules` are breaking changes. We should consider using the utils not in the latest version. After the merge, this will be an issue in particular for all active WEKit projects.

Reply:
just thought why not being backwards compatible (like contentful) and also allow the page type
```
 {{ if or (eq $content_type "page") (hasPrefix $content_type "t-") }}
```

or we could also just make the "page" types configurable in the hugo config. So we don't need to hardcode any opinionated content type names

Reply:
I have mixed feelings about this. 😃 I think it's good to be backwards compatible in the first place. On the other hand, we wanted to remain simple and maintainable. 

Similar to the point of having page types configurable. Nice, but... 🤔 And we actually want to provide best practices so that decisions can be made more easily.

Reply:
We definitely provide best practices but this could also be done less "hard wired" ;)
And concerning the maintainability... this changes happen in the core utils which should just work so that the vast majority of devs won't even take a look at the implementation details

Reply:
And after reading this https://go.dev/blog/v2-go-modules we should definitely try to stay backwards compatible if possible :)

> Breaking changes to a v1+ module should always happen in a new, vN+1 module. When a new module is released, it means additional work for the maintainers and for the users who need to migrate to the new package. Maintainers should therefore validate their APIs before making a stable release, and consider carefully whether breaking changes are really necessary beyond v1.

Reply:
We are now backwards compatible with the page content type.